contribution.md

```diff
@@ -0,0 +1,54 @@
+# Synthesis Contribution Guide
+Synthesis is 100% open source and relies on the FIRST community to help make it better. The Synthesis Contribution Guide suggests ways in which you can get involved through development and non-development avenues. 
+
+# Why Contribute? Benefits to Contributing
+* Prepare for an internship - share your contributions when applying to the [Synthesis Summer Internship.](https://synthesis.autodesk.com/internship.html)
+* Add your contributions to Synthesis: An Autodesk Technology to your portfolio
+* Meet other members in the FIRST community
+* Get involved and learn more about Autodesk products
+
+# How to Contribute
+### Found a bug? Have an idea for a feature?
+Submit your bug report or idea to our [GitHub issues](https://github.com/Autodesk/synthesis/issues/new/choose).
+
+*A Note to Developers* When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change. This  way, we can ensure that there is no overlap between contributions and internal development work. You may contact us at frc@autodesk.com.
+
```

Comment:
Probably mention that this is really only necessary when making a large change. Perhaps also add something like: Feel free to send us a [pull request](https://github.com/Autodesk/synthesis/pulls).

Reply:
Just added those suggestions. Thanks!